src/main/java/employeetracker/model/employee/UniqueEmployeeList.java

```diff
@@ -313,9 +321,9 @@ public String getAvgTenure() {
         } catch (Exception e) {
             System.out.println(e.toString());
         }
-        avgTensure = totalMillies / internalList.size();
-        int totalYears = (int) (avgTensure / yearMiniSec);
-        int days = (int) (avgTensure / dayMiniSec - (totalYears * 365));
+        averageTensure = totalMillies / internalList.size();
```

Comment:
Perhaps it should be averageTenure instead of averageTensure? Tensure is not an english word